rubicon/objc/runtime.py

```diff
@@ -383,6 +387,18 @@ class objc_property_attribute_t(Structure):
 libobjc.objc_getProtocol.restype = objc_id
 libobjc.objc_getProtocol.argtypes = [c_char_p]
 
+# id objc_initWeak(id *object, id value)
+libobjc.objc_initWeak.restype = objc_id
+libobjc.objc_initWeak.argtypes = [POINTER(objc_id), objc_id]
```

Comment:
Minor thing: this function is never used, so we don't need to declare it.

Reply:
True. It's a leftover when I was trying to figure out LLVMs generated calls.